### DISCUSSION TITLE
Complex docker-compose setup

### DISCUSSION THREAD

**Discussion by delanym** [OpenSignLabs/OpenSign#545] [General]
Hi. Its not clear which env vars are actually needed
https://docs.opensignlabs.com/docs/self-host/docker/run-locally#information-about-env-variables-which-are-used-to-setup-opensign-with-docker-on-localhost

Do I have to deploy on DigitalOcean or AWS?
Do I really need mailgun?
Why is there a MONGODB_URI if this is part of the container stack?

I just want to see how this application looks without doing a whole bunch of setup.

**Comment by andrew-opensignlabs** [comment]
You can install it on digitalocean from below link -

https://docs.opensignlabs.com/docs/self-host/One-click-Install/DigitalOcean

You dont need mailgun. You can use any smtp server.

MongodDB uri allows you to configure the app to connect to an external mongodb.

You can also sign up at https://app.opensignlabs.com using coupon code "FREEBETA" to get the cloud hosted version for free. The code is valid only till tomorrow.

**Comment by delanym** [comment]
So I have to have DigitalOcean. Then its not for me

**Comment by cipherw0lf** [comment]
Yes. Its sickening, been 10 hours. /api/app simply reaches the react web app instead of server api. I even tried public url for server url - but it just times out. On digital ocean it magically works.
